### PR TITLE
Update Clear Command

### LINKED ISSUE #127: [PE-D] Graph is not updated after clear command
After testing the plotting of the graph after adding exercises (which works, good job!). I tried testing the `clear` command. While the `clear` command works as expected, the graph was not updated accordingly (see screenshot below)

Though it is a minor issue, you may want to consider updating the graph to match the updated data in the application when using the `clear` command as well :)

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: ChooJiaXin/ped#4

### PR DESCRIPTION
**Pull request by Nahoyhp**
- To clear Template
- To clear graph

### CODE REVIEW

File: src/main/java/seedu/address/model/ExerciseBook.java [hunk 1]
```diff
@@ -51,16 +51,28 @@ public void setExercises(List<Exercise> exercises) {
         this.exercises.setExercises(exercises);
     }
 
+    /**
+     * Replaces the contents of the person list with {@code persons}.
+     * {@code persons} must not contain duplicate persons.
```
Comment (Liu-2001) [inline src/main/java/seedu/address/model/ExerciseBook.java]: should be exercises

File: src/main/java/seedu/address/model/ExerciseBook.java [hunk 2]
```diff
@@ -34,7 +34,7 @@ public ExerciseBook() {
     }
 
     /**
-     * Creates an AddressBook using the Persons in the {@code toBeCopied}
+     * Creates an ExerciseBook using the Exercises in the {@code toBeCopied}
```
Comment (aaronlwm) [inline src/main/java/seedu/address/model/ExerciseBook.java:39]: capitalised?

File: src/main/java/seedu/address/model/ExerciseBook.java [hunk 3]
```diff
@@ -44,53 +44,61 @@ public ExerciseBook(ReadOnlyExerciseBook toBeCopied) {
...
-    //// person-level operations
+    /**
+     * Reset all the data inside ExerciseBook.
+     */
+    public void resetAllData() {
+        exercises.resetAll();
+        TemplateList.reset();
+    }
+
+    //// exercise-level operations
```
Comment (aaronlwm) [inline src/main/java/seedu/address/model/ExerciseBook.java]: double /

File: src/main/java/seedu/address/model/ExerciseBook.java [hunk 4]
```diff
@@ -44,53 +44,61 @@ public ExerciseBook(ReadOnlyExerciseBook toBeCopied) {
...
 
     /**
-     * Returns true if a person with the same identity as {@code person} exists in the address book.
+     * Returns true if a exercise with the same identity as {@code exercise} exists in the exercise book.
```
Comment (aaronlwm) [inline src/main/java/seedu/address/model/ExerciseBook.java]: Capitalised?

Comment (Nahoyhp) [reply src/main/java/seedu/address/model/ExerciseBook.java]: Fixed

File: src/main/java/seedu/address/model/ExerciseBook.java (hunk 3 above)
Comment (Nahoyhp) [reply src/main/java/seedu/address/model/ExerciseBook.java]: Fixed

**Pull request merged**